packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -77,8 +81,10 @@ class CupertinoNavigationBar extends StatelessWidget implements ObstructingPrefe
     this.automaticallyImplyLeading: true,
     this.middle,
     this.trailing,
+    this.borderColor: _kDefaultNavBarBorderColor,
     this.backgroundColor: _kDefaultNavBarBackgroundColor,
     this.actionsForegroundColor: CupertinoColors.activeBlue,
+    this.adjustSystemUiOverlayStyle : _kDefaultNavBarAdjustSystemUIOverlayStyle,
```

Comment:
you can just inline the `true` here, it'll make the code clearer in the documentation

Reply:
nit: no space before `:`